LondonBritishLibrary/orient/BLorient8824.xml

```diff
@@ -0,0 +1,417 @@
+        <fileDesc>
+            <titleStmt>
+                <title>Miracles of Jesus</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8824</idno>
+                        <altIdentifier><idno>Strelcyn 16</idno></altIdentifier>
+                        <altIdentifier><idno>Curzon, Cat. E6</idno></altIdentifier>
+                        <altIdentifier><idno>Parham 60</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="2ra" to="179va"/>
+                            <title ref="LIT2382Taamme"/>  
+                            <note>Numbered from 1 to 42 plus two unnumbered miracles.</note>
+                                <msItem xml:id="ms_i1.1">
+                                    <locus from="2ra"/>
+                                    <title ref="LIT6396MJIntroduction"/>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/>
+                                        ፩አምላ<supplied reason="undefined" resp="PRS8999Strelcyn">ክ፡</supplied> ፩ተአምሪሁ፡ ለእግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡
+                                        ኢየሱስ፡ ክርስቶስ፡ ሣህሉ፡ ወምሕረቱ፡ ወስእለተ፡ እሙ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ወልደ፡ ትንሣኤ፡ ወምስለ፡ ወለተ፡ ሩፋኤል፡ ለዓለመ፡ ዓለም፡ አሜን። ለዝንቱ፡ 
+                                        ምሥጢር፡ አምላካዊ፡ ነገሮ፡ እግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ኢየሱስ፡ ክርስቶስ፡ በረድኡ፡ ወሐዋርያሁ፡ ዮሐንስ፡ ወልደ፡ ዘብዴዎስ፡ እምቅድመ፡
+                                        ዕርገቱ፡ ለእግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክ<cb n="2rb"/>ርስቶስ፡ ውስተ፡ ሰማያት፡ ዘኢተፈለጠ፡ እምአቡሁ፡ ወእመንፈሱ፡ ቅዱስ፡ አሐተ፡ ሰዓተ፡ 
+                                        ወኢከመ፡ ቅፅበተ፡ ዓይን፤ ኀረየ፡ ፲ወ፪አርድእተ፡ ወረሰዮሙ፡ ጻድቃነ፡ ወሐዋርያተ። </incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2">
+                                    <locus from="10va"/>
+                                    <title ref="LIT00000000000000000">On the Conception of Jesus</title>
```

Comment:
this could be the miracle otherwise transmitted with the miracles of Mary LIT5382MiracleConception but of course it is impossible to say without the text

Reply:
I will add LIT5382MiracleConception in a note as probable text.